common/test/acceptance/tests/lms/test_lms_edxnotes.py

```diff
@@ -872,18 +872,27 @@ def assert_page(note, usage_id, view):
 
         self._add_default_notes()
         self.notes_page.visit()
+        # visiting the page results in an ajax request to fetch the notes
+        self.notes_page.wait_for_ajax()
```

Comment:
FWIW my preference is for it to go on the page object. This is because some other dev could write another test and they might not know to put the wait_for_ajax into **their** test and it'll end up being "flaky".